src/ontology/mondo-edit.obo

```diff
@@ -93307,7 +93307,7 @@ xref: ICD10CM:K00.0 {source="Orphanet:99798/ntbt", source="Orphanet:99798", sour
 xref: ICD9:520.0 {source="DOID:0050591"}
 xref: MESH:D000848 {source="DOID:0050591", source="MONDO:directSiblingOf"}
 xref: OMIMPS:106600 {source="MONDO:cjm", source="MONDO:equivalentTo"}
-xref: Orphanet:2227 {source="MONDO:equivalentTo", source="DOID:0050591"}
+xref: Orphanet:2227 {source="MONDO:equivalentObsolete", source="DOID:0050591"}
 xref: Orphanet:99798 {source="MONDO:equivalentTo", source="MONDO:preferredExternal", source="DOID:0050591"}
```

Comment:
preferredExternal is now lo longer necessary but could be handled in a 2nd pass

Reply:
https://github.com/monarch-initiative/mondo/issues/6339